example/example.dart

```diff
@@ -8,19 +8,21 @@
 /// Author: Max Korbel <max.korbel@intel.com>
 ///
 
-// ignore_for_file: avoid_print
+// Import the standard library for I/O.
+import 'dart:io';
```

Comment:
See other comment on PR

Reply:
Agree. I didn't think from that position. I'll revert `print` back and add an explanatory comment.